crates/ibc/src/core/context.rs

```diff
@@ -255,7 +255,7 @@ mod val_exec_ctx {
 
         /// Returns a natural number, counting how many clients have been created thus far.
         /// The value of this counter should increase only via method `ClientKeeper::increase_client_counter`.
-        fn client_counter(&self) -> Result<u64, ContextError>;
+        fn generate_client_identifier(&self) -> Result<u64, ContextError>;
```

Comment:
Note that we have an issue (#14) that tracks this. But also if the function were to be named `generate_client_identifier`, I would expect it to return a `ClientId` as opposed to an "id counter".

Can we make that change in another PR that addresses #14, in order to have self-contained PRs?

Reply:
Didn't notice that issue. 
Sure. Will do so